lib/main.rb

```diff
@@ -0,0 +1,52 @@
+require_relative './app'
+require_relative './book'
+def main
+  puts 'Welcome to school library app!'
+  app = App.new
+
+  choices = 1
+  until choices.eql? 7
+    puts 'Please choose an optional by enter in a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
```

Comment:
While trying to create a rental for any number other than 0, your application runs into an infinite loop. Kindly fix the issue. You can view [this](https://www.loom.com/share/776d33a9008d4434a71944fa5efcc665) video for further clarification, where i have tried to explain the problem in detail.

Reply:
Hello @muneebulrehman, Thank you for your feedback. I think I already implement that. When you try to enter a number outside the range, it asks you to enter again. 
![image](https://user-images.githubusercontent.com/30173722/201346110-980a1479-08b0-4265-a6e6-aada52f93668.png)